apps/docs/content/components/Pillar/index.mdx

```diff
@@ -17,34 +20,35 @@ export default ComponentLayout
 
 ## Usage
 
-Use pillars to group similar features or benefits together to easily scan and read. Pillars display content in a familiar and recognizable style.
+Use pillars to display a collection of features or benefits, making them easier to scan and read. Pillars present content in a familiar and recognizable style.
 
 ### Stacked
 
-Pillars should be stacked to display a list of items that are related to each other. Use consistent pillar sizes within a row or group. A group or a row of content pillars is more visually appealing when the widths and heights of all elements match, so keep the content lenght of each pillar consistent.
+Stack multiple pillars to create a collection of items. Ensure that all the items in a group have consistent content lengths so that their width and height match, making the content more visually appealing. When displaying a visual, create a balanced layout by using the same type for each pillar, either an icon or an image.
 
 <DoDontContainer>
   <Do>
     <img src="https://github.com/primer/brand/assets/912236/cd9679ab-cfa0-4c6d-8a55-21d16668f1ca" />
     <Caption>
-      Use a similar content size to achieve a visually balanced layout.
+      Use similar content lengths to achieve a visually balanced layout.
     </Caption>
   </Do>
   <Dont>
     <img src="https://github.com/primer/brand/assets/912236/cd6e471f-98cf-4c70-801f-c0aeb4d2bdf5" />
     <Caption>
-      Don't use different length sizes for the title or the description.
+      Avoid varying content lengths for the title and description.
     </Caption>
   </Dont>
 </DoDontContainer>
 
-If displaying an icon, do so for all cards to achieve a visually balanced layout.
+Use pillars with images only for groups of a maximum of three to six items. Avoid stacking multiple sections that each contain groups of six pillars.
```

Comment:
Recommend drawing more attention to this by wrapping in a warning Note. I think it's a little lost right now. Could also benefit from a visual do/dont example if you can do it.

```suggestion
<Note variant="warning">
Use pillars with images only for groups of a maximum of three to six items. Avoid stacking multiple sections that each contain groups of six pillars.
</Note>
```